librz/include/rz_cmd.h

```diff
@@ -91,6 +91,7 @@ typedef enum {
 	RZ_OUTPUT_MODE_LONG_JSON = 1 << 6,
 	RZ_OUTPUT_MODE_TABLE = 1 << 7,
 	RZ_OUTPUT_MODE_QUIETEST = 1 << 8,
+	RZ_OUTPUT_MODE_GRAPH = 1 << 9,
```

Comment:
Nice addition but there could be multiple types of graph - e.g. native, dot one, interactive, and so on. While it's less relevant in this PR, we should think this over nevertheless, possibly at some later stage. But preferably before the release.

Reply:
Yeah, would be nice. I can attempt to add it here. If it is too much struggle we can still do it later.

Reply:
Sounds good to me.